src/page-weekly-snapshot/projectionChartUtils.tsx

```diff
@@ -0,0 +1,187 @@
+import * as dateFns from "date-fns";
+import ndarray from "ndarray";
+
+import {
+  EpidemicModelInputs,
+  getLocaleDefaults,
+  RateOfSpread,
+  totalConfirmedCases,
+  totalConfirmedDeaths,
+} from "../impact-dashboard/EpidemicModelContext";
+import { countCasesForDay } from "../impact-dashboard/ImpactProjectionTableContainer";
+import { NUM_DAYS } from "../infection-model";
+import { calculateCurves, curveInputsFromUserInputs } from "../infection-model";
+import { getAllValues, getColView } from "../infection-model/matrixUtils";
+import { seirIndex } from "../infection-model/seir";
+import { LocaleData } from "../locale-data-context";
+import { ModelInputs } from "../page-multi-facility/types";
+import { combineFacilitiesProjectionData } from "../page-response-impact/responseChartData";
+
+export function getCaseCount(curveData: ndarray<number>) {
+  const cases = [];
+  for (let day = 0; day < curveData.shape[0]; ++day) {
+    const caseCountForDay = countCasesForDay(curveData, day);
+    cases.push(caseCountForDay);
+  }
+  return cases;
+}
+
+export const today = dateFns.endOfToday();
```

Comment:
I might be way wrong about this, but will these lines only be executed and these constants be defined and exported only once when the app first starts and the export graph is initialized? That would mean that these exports would become outdated when we passed into the next day after the first deploy that includes this code. Let me know if I just misunderstand and if these are actually being lazily initialized or something!

Reply:
Yeah, that makes sense...I think it could cause an issue if the app isn't reloaded, and I would actually want a function then to return endOfToday() whenever we it's being used, or just use the `dateFns.endOfToday()` directly.